### PR TITLE
UPGRADE.md fixes

### PR DESCRIPTION
**Pull request by boris-brtan**
| Q             | A
| ------------- | ---
|Description, reason for the PR| improvement of UPGRADE from beta6 into stable based on [shopsys/demoshop#39]
|New feature| No 
|[BC breaks](https://github.com/shopsys/shopsys/blob/master/docs/contributing/backward-compatibility-promise.md)| No 
|Fixes issues| - 
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: docs/upgrade/UPGRADE-v7.0.0.md [hunk 1]
```diff
@@ -54,7 +54,13 @@ if you want to have products data exported to Elasticsearch after `build-demo` t
             - { resource: services/commands.yml }
     +       - { resource: services/data_fixtures.yml }
     ```
-    - change namespaces of data fixtures from `Shopsys/FrameworkBundle/DataFixtures` to `Shopsys/ShopBundle/DataFixtures` (in tests and *.yml configurations)
+    - change namespaces of data fixtures from `Shopsys\FrameworkBundle\DataFixtures` to `Shopsys\ShopBundle\DataFixtures` (in tests and *.yml configurations)
+        - check correctness of registered DataFixture services based on the state of [this config file](https://github.com/shopsys/shopsys/blob/v7.0.0/project-base/src/Shopsys/ShopBundle/Resources/config/services.yml)
```
Comment (DavidOstrozlik) [inline docs/upgrade/UPGRADE-v7.0.0.md]:
```suggestion
        - check correctness of registered DataFixture services based on the state of [services.yml](https://github.com/shopsys/project-base/blob/v7.0.0/src/Shopsys/ShopBundle/Resources/config/services.yml) of [shopsys/project-base] repository
```

File: docs/upgrade/UPGRADE-v7.0.0.md [hunk 2]
```diff
@@ -54,7 +54,13 @@ if you want to have products data exported to Elasticsearch after `build-demo` t
...
+    - change the value of data fixtures resource folder in `paths.yml`
```
Comment (DavidOstrozlik) [inline docs/upgrade/UPGRADE-v7.0.0.md]:
```suggestion
    - change the value of data fixtures resource folder in `app/config/paths.yml`
```

File: UPGRADE.md [hunk 3]
```diff
@@ -27,12 +27,10 @@ Follow instructions in a [monorepo upgrade guide](docs/contributing/upgrading-mo
     * *(MacOS, Windows only)* run `docker-sync stop`
     * *(MacOS, Windows only)* run `docker-sync clean` so your volumes will be removed
     * follow upgrade notes in a *Infrastructure* section (related with `docker-compose.yml`, `Dockerfile`, docker containers, `nginx.conf`, `php.ini`, etc.)
-    * change all the microservices image versions in your `docker-compose.yml` to version you are upgrading to
-        eg. `image: shopsys/microservice-product-search:v7.0.0-beta1`
     * *(MacOS, Windows only)* run `docker-sync start` to create volumes  
-    * run `docker-compose up -d --build --force-recreate` to start application again
+    * run `docker-compose up -d --build --force-recreate --remove-orphans` to start application again
```
Comment (DavidOstrozlik) [inline UPGRADE.md]: is it really necessary to add the --remove-orphans? Is it because of removal of microservices?

Comment (boris-brtan) [reply UPGRADE.md]: yes

**Pull request merged**